cmd/build.go

```diff
@@ -296,14 +358,15 @@ func normalizeTarget(target string) string {
 	return strings.TrimLeft(target, "/")
 }
 
-func runGenerator(mode string, flags []string) generatorOutput {
+func runGenerator(input generatorInput) generatorOutput {
```

Comment:
Instead of a half-populated input this can take the individual fields it expects? (Just BuildFlags, RunArgs and CompletionOnly bool)